site/en/blog/google-summer-of-code-and-chrome-extensions/index.md

```diff
@@ -0,0 +1,103 @@
+
+## Wrapping Up
+
+If you ask me what GSoC has brought me: first of all, of course, is understanding the knowledge related to browser extension APIs. By reading the documentation and writing new samples, I gained a detailed understanding of various browser extension APIs.
+
+Additionally, I have acquired many skills I had little exposure to before being accepted for GSoC, such as following a comprehensive code review process and creating PRDs. I have also started using GitHub actions to automate workflows and learned how to use Babel to traverse all extension samples and track the APIs used. Moreover, I had my first experience with the Nunjucks template engine. These abilities are essential for programming, and I am delighted that I learned them in advance. Participating in open-source communities brings me joy.
+
+This summer has given me many indelible memories. Coding makes me highly content. I have encountered patient mentors who guided me and taught me interesting knowledge, enriching my experience.
+
+The knowledge in the field of computer science is endless. When I compared myself with others, I realized my understanding of open source was only the tip of the iceberg. However, every little bit helps. I believe GSoC is a great starting point for participating in open source.
+
+I also hope my experience can provide some reference for future participation in open-source activities.
+
+Here are some of my suggestions for anyone wanting to participate in Google Summer of Code:
+
+1. Choose an idea that suits you: In my opinion, when looking for ideas, interest should be your priority, followed by skills. This strategy will allow you to maintain enthusiasm throughout the contribution process.
+2. Quality over quantity: You can submit many proposals to increase the chances of being accepted for GSoC. However, I believe that quality is probably more critical because human energy is limited. Focusing on two or three ideas may be more helpful in the end.
+3. Don't be afraid of new things: Don't hesitate to try new technologies you haven't experienced before in a project. You have nearly a month of community bonding to familiarize yourself with unfamiliar technologies and a whole summer to implement them. Just go for it!
+4. Get acquainted with the project in advance: For example, read through development documentation, look at previous commit history and issues, and skim through past pull requests. Getting an overall understanding of the project in advance will help you write a more comprehensive proposal.
+5. Maintain communication: Regularly communicate with your mentor, and don't hesitate to ask questions when confronting problems, whether before applying or after being accepted. Most mentors are willing to help you solve problems; this way, they can also keep track of your progress.
+
+Thank you for reading!
```

Comment:
```suggestion
Thank you for reading.
```

Reply:
I would leave the "!"... seems to flow better with the tone of the blog :)

Reply:
I want to keep this comment. This is actually an incorrect use of the exclamation point.